### PR TITLE
Update the example init file

### PR DESCRIPTION
**Pull request by PeridexisErrant**
After updating the extra init file in my pack, I realised that many of
the changes could be merged into the standard init.

This enables a few more UI improvements, adds neglected keybindings, and
removes the roses-init and binpatches sections (third-party only and
obsolete respectively).

### CODE REVIEW

File: dfhack.init-example [hunk 1]
```diff
@@ -177,6 +181,7 @@ tweak import-priority-category
 # Misc. UI tweaks
 tweak block-labors              # Prevents labors that can't be used from being toggled
 tweak civ-view-agreement
+tweak eggs-fertile
```
Comment (lethosor) [inline dfhack.init-example]: I have mixed feelings about enabling this by default. I said that I considered it cheating when it came up for discussion when I added it, and most of the other tweaks enabled by default are bugfixes or improvements.

Comment (PeridexisErrant) [inline dfhack.init-example]: I think of it as a UI improvement, but if you want me to I can remove it.

File: dfhack.init-example [hunk 2]
```diff
@@ -223,7 +228,7 @@ enable \
 # You can comment out the extension of a line.
 
 # enable mouse controls and sand indicator in embark screen
-embark-tools enable sand mouse
+embark-tools enable sticky sand mouse
```
Comment (lethosor) [inline dfhack.init-example]: This one's really a personal preference, in my opinion. It also has issues with selecting uninhabitable sites, which vanilla DF avoids.

File: dfhack.init-example [hunk 3]
```diff
@@ -232,14 +237,11 @@ embark-tools enable sand mouse
 # write extra information to the gamelog
 modtools/extra-gamelog enable
 
+# extended status screen (bedrooms page)
+enable gui/extended-status
```
Comment (lethosor) [inline dfhack.init-example]: I really ought to expand this one eventually...

File: dfhack.init-example (hunk 1 above)
Comment (lethosor) [inline dfhack.init-example]: I could make it possible to toggle in the UI, I suppose (like `tweak hide-priority`).

**Pull request merged**